### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Rinbo**
## Setup testing environment
PT: https://www.pivotaltracker.com/story/show/160584192

Install Rspec and Cucumber by adding the appropriate gems and then running the following commands.
~~~
bundle exec rails generate rspec:install
bundle exec rails generate cucumber:install
~~~
We also migrated the database but we had to modify the migration version in the schema files to 4.2

### CODE REVIEW

File: features/support/env.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+#   Before('not @no-txn', 'not @selenium', 'not @culerity', 'not @celerity', 'not @javascript') do
+#     DatabaseCleaner.strategy = :transaction
+#   end
+#
+
+# Possible values are :truncation and :transaction
+# The :transaction strategy is faster, but might give you threading problems.
+# See https://github.com/cucumber/cucumber-rails/blob/master/features/choose_javascript_database_strategy.feature
+Cucumber::Rails::Database.javascript_strategy = :truncation
+
```
Comment (faraznaeem) [inline features/support/env.rb]: clean out code

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  # order dependency and want to debug it, you can fix the order by providing
+  # the seed, which is printed after each run.
+  #     --seed 1234
+  config.order = :random
+
+  # Seed global randomization in this process using the `--seed` CLI option.
+  # Setting this allows you to use `--seed` to deterministically reproduce
+  # test failures related to randomization by passing the same `--seed` value
+  # as the one that triggered the failure.
+  Kernel.srand config.seed
```
Comment (faraznaeem) [inline spec/spec_helper.rb]: Please remove comments

**Review by faraznaeem — CHANGES_REQUESTED** [review]

**Comment by faraznaeem** [comment]
You don't need to specify the exact changes to the PR description, we can see that in the files changed.
Also this PR is to big. It should have been divided into two PRs one for Cucumber and one for Rspec.
It's fine this time but in the future make sure to divide them up
The story should also be added to the description or a link to the specific story to PT or Waffle.

**Comment by Rinbo** [comment]
We haven't split the PR into two different ones but we will keep this in mind the next time we do a PR. Hope that is ok.

File: features/signup_to_account.feature [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    Given I am on the landing page
+    When click on the "Sign up" button
+    Then I fill in "Name" with "Bob"
+    And I fill in "Email" with "bob@example.com"
+    And I fill in "Password" with "foobar"
+    And I fill in "Password confirmation" with "foobar"
+    When I click "Create"
+    Then I should be on the landing page
+    And I should see a message "What message?"
+
```
Comment (faraznaeem) [inline features/signup_to_account.feature]: Please delete this feature file it should go in the signup for account PR not this one.

**Review by faraznaeem — CHANGES_REQUESTED** [review]

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**